CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+- `list-context.image-list` has been refactored to use the `image-list` with a `list-context`
+
+### Added
+- `image-list` is now a simple list of images that can be editable from the admin panel 
```

Comment:
This can go below `Changed` too. No need for this `Added` section

Reply:
@victorhmp That is a new component, it should be under Added